### DISCUSSION TITLE
Pytest is not creating physical entries in local testserver.

### DISCUSSION THREAD

**Discussion by yashvarshney003** [pytest-dev/pytest#10989] [General]
This is pytest.ini 
```
[pytest]
python_files = tests.py test_*.py *_tests.py
addopts = --no-migrations
testpaths =
    tests
```
When I run test cases by command pytest i can found database is created on my local postgresql server but no entry found in any table when i stop the test cases by pdb.

**Comment by RonnyPfannschmidt** [comment]
Without more context no sensible answer is possible

**Comment by yashvarshney003** [comment]
@RonnyPfannschmidt  Please let me know what more information is needed from my side.
